scripts/camViewer

```diff
@@ -295,17 +302,27 @@ if [ $CAMNUM -eq 0 ]; then
 	CAMNAME=xcs_yag2
     fi
 
-
 fi
-    
 export EPICS_CA_MAX_ARRAY_BYTES=24000000
-echo working with camera of name: $CAMNAME
-
+echo working with camera of name: "$CAMNAME"
 
 name_to_pv "$CAMNAME" "$hutch"
 
 if [ $MAINSCREEN -gt 0 ]; then
-    echo 'try to get the main screen for: ' $CAMNAME
+    tmp=/tmp/cv.$$
+    list_cams "$hutch" >$tmp
+    c=$(wc <$tmp | grep "$CAMNAME" $tmp | awk '{print $1}')
+    if [ "$c" == '' ]; then
+        echo "Camera name not found in hutch. To see which cameras you can access from your current machine use the '-l/--list' option."
+        sleep 3
+    elif [ "$c" == "$CAMNAME" ] && [ "$hutch" != "$(get_info --gethutch)" ] ; then
+        echo "To use camera controls, you must be on the same hutch machine as your camera." 
+        echo "Opening read-only main screen..."
+        sleep 3
+    
+    rm -f $tmp
+    fi    
```

Comment:
Two things here:
- It looks like this snippit is copy/pasted from the `name_to_pv` function? Is there a reason why that function couldn't have been re-used?
- This isn't mentioned in the PR text, is this just an incidentally bundled fix?